build/AzurePipelinesTemplates/MUX-BuildProject-Steps.yml

```diff
@@ -0,0 +1,55 @@
+parameters:
+  solutionPath: ''
+  nugetConfigPath: ''
+  appxPackageDir: '$(build.artifactStagingDirectory)\$(buildConfiguration)\$(buildPlatform)\AppxPackages'
+  buildOutputDir: '$(Build.SourcesDirectory)\BuildOutput'
+  copyFilesToStaging: false
+  publishPath: '$env:Build_ArtifactStagingDirectory'
+
+steps:
+  - task: CmdLine@1
+    displayName: 'Display build machine environment variables'
+    inputs:
+      filename: 'set'
+
+  - task: powershell@2
+    inputs:
+      targetType: filePath
+      filePath: build\Install-WindowsSdkISO.ps1
+      arguments: 17763
+    displayName: 'Install RS5 SDK (17763)'
+
+  - task: NuGetToolInstaller@0
+    displayName: 'Use NuGet 4.9.1'
+    inputs:
+      versionSpec: 4.9.1
+        
+  - task: 333b11bd-d341-40d9-afcf-b32d5ce6f23b@2
```

Comment:
>333b11bd-d341-40d9-afcf-b32d5ce6f23 [](start = 10, length = 35)

what the UUID mean?

Reply:
It's some Azure Pipelines task -- it's what shows up in the website if you click "View YAML" of this task.

Reply:
It's the NuGet task. Usually you can just refer to the task by name ('NuGet'), but something is up with the Microsoft account that causes this to fail due to an ambiguous reference. I don't know why the Microsoft Azure DevOps accounts has two tasks installed named 'NuGet', but this is the best work-around. The displayName param ensures that this task shows up as something readable in Azure Pipelines.